packages/flutter/example/pubspec.yaml

```diff
@@ -12,21 +14,19 @@ dependencies:
   cupertino_icons: ^1.0.2
   parse_server_sdk_flutter:
     path: ../
-  path_provider: ^2.0.1
-  sembast: ^3.0.0+4
-  shared_preferences: ^2.0.5
 
-dependency_overrides:
-  # Override to local mono-repo path so devs can test this repo
-  # against changes that they're making to other mono-repo packages.
-  parse_server_sdk:
-    path: ../../dart
+  path: ^1.8.3
+  path_provider: ^2.0.15
+  sembast: ^3.4.4
+  shared_preferences: ^2.1.1
```

Comment:
Add the dependency_overrides section. Without it, the Flutter SDK will actually get the Dart SDK from pub!

```
dependency_overrides:
  # Override to local mono-repo path so devs can test this repo
  # against changes that they're making to other mono-repo packages.
  parse_server_sdk:
    path: ../../dart
```

Reply:
Sure